tests/system/test_asyncpg_connection.py

```diff
@@ -63,6 +64,8 @@ async def create_sqlalchemy_engine(
             The database user's password, e.g., secret-password
         db (str):
             The name of the database, e.g., mydb
+        ip_type (str):
+            The IP type of the Cloud SQL instance. Can be one of "public", "private", or "psc".
```

Comment:
nit: let's update this to not go over 80 chars per line

```suggestion
            The IP type of the Cloud SQL instance to connect to. Can be one of "public",
            "private", or "psc".
```

Reply:
Done.